operator/main.py

```diff
@@ -211,10 +212,13 @@ def validate_volume_request(obj):
                       " specified")
         return False
 
+    # should'nt this come under disperse volume?? Not replica??
```

Comment:
Are these questions to reviewers? :P

Reply:
Sorry I had to remove this before updating here !!